src/_posts/2018-08-XX-review-input-streaming-connectors.md

```diff
@@ -0,0 +1,224 @@
+
+<table>
+  <tr>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td><strong>Apache Beam</strong>
+   </td>
+   <td><strong>Apache Spark DStreams</strong>
+   </td>
+  </tr>
+  <tr>
+   <td rowspan="2" >File Systems
+   </td>
+   <td>Local
+   </td>
+   <td><a href="https://beam.apache.org/documentation/sdks/pydoc/2.5.0/apache_beam.io.textio.html">io.textio</a>
+   </td>
+   <td><a href="http://spark.apache.org/docs/latest/api/python/pyspark.streaming.html#pyspark.streaming.StreamingContext.textFileStream">textFileStream</a>
+   </td>
+  </tr>
+  <tr>
+   <td>HDFS
+   </td>
+   <td><a href="https://beam.apache.org/documentation/sdks/pydoc/2.5.0/apache_beam.io.hadoopfilesystem.html">io.hadoopfilesystem</a>
+   </td>
+   <td><a href="https://spark.apache.org/docs/latest/api/java/org/apache/spark/SparkContext.html#hadoopConfiguration--">hadoopConfiguration</a> (Access through <code>sc._jsc</code> with Py4J)
+and <a href="http://spark.apache.org/docs/latest/api/python/pyspark.streaming.html#pyspark.streaming.StreamingContext.textFileStream">textFileStream</a>
+   </td>
+  </tr>
+  <tr>
+   <td rowspan="2" >Object stores
+   </td>
+   <td>Google Cloud Storage
+   </td>
+   <td><a href="https://beam.apache.org/documentation/sdks/pydoc/2.5.0/apache_beam.io.gcp.gcsio.html">io.gcp.gcsio</a>
+   </td>
+   <td rowspan="2" ><a href="http://spark.apache.org/docs/latest/api/python/pyspark.streaming.html#pyspark.streaming.StreamingContext.textFileStream">textFileStream</a>
+   </td>
+  </tr>
+  <tr>
+   <td>S3
+   </td>
+   <td>N/A
+   </td>
+  </tr>
+  <tr>
+   <td rowspan="3" >Messaging Queues
+   </td>
+   <td>Kafka
+   </td>
+   <td>N/A
+   </td>
+   <td><a href="http://spark.apache.org/docs/latest/api/python/pyspark.streaming.html#pyspark.streaming.kafka.KafkaUtils">KafkaUtils</a>
+   </td>
+  </tr>
+  <tr>
+   <td>Kinesis
+   </td>
+   <td>N/A
+   </td>
+   <td><a href="http://spark.apache.org/docs/latest/api/python/pyspark.streaming.html#module-pyspark.streaming.kinesis">KinesisUtils</a>
+   </td>
+  </tr>
+  <tr>
+   <td>Cloud Pub/Sub
+   </td>
+   <td><a href="https://beam.apache.org/documentation/sdks/pydoc/2.5.0/apache_beam.io.gcp.pubsub.html">io.gcp.pubsub</a>
+   </td>
+   <td>N/A
+   </td>
+  </tr>
+  <tr>
+   <td>Other
+   </td>
+   <td>Custom receivers
+   </td>
+   <td><a href="https://beam.apache.org/documentation/sdks/python-custom-io/">BoundedSource and RangeTracker</a>
+   </td>
+   <td>N/A
+   </td>
+  </tr>
+</table>
+
+## Connectors for other languages
+
+### **Scala**
+
+Since Scala code is interoperable with Java and therefore has native compatibility with Java libraries (and vice versa), you can use the same Java connectors described above in your Scala programs. Apache Beam also has a [Scala SDK](https://github.com/spotify/scio) open-sourced [by Spotify](https://labs.spotify.com/2017/10/16/big-data-processing-at-spotify-the-road-to-scio-part-1/).
```

Comment:
s/SDK/API as referred in the scio github page. Probably more clear to say Spotify has a Scala API on top of Apache Beam.